package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "samepage",
-  "version": "0.74.0",
+  "version": "0.74.1",
```

Comment:
as an fyi, version bumps are only needed when there are changes to the `package/*` directory